recipe/migrations/cuda_112_ppc64le_aarch64.yaml

```diff
@@ -0,0 +1,37 @@
+migrator_ts: 1645421738
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: true
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  wait_for_migrators:
+    - aarch64andppc64leaddition
+  commit_message: "Rebuild for cuda for ppc64le and aarch64"
+
+c_compiler_version:            # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+cxx_compiler_version:          # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+fortran_compiler_version:      # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+
+cuda_compiler_version:         # [ppc64le or aarch64]
+  - 11.2                       # [ppc64le or aarch64]
+
+cudnn:                  # [ppc64le or aarch64]
+  - 8                   # [ppc64le or aarch64]
```

Comment:
Can we set this to undefined for now? The PR to support both platforms is still blocked: https://github.com/conda-forge/cudnn-feedstock/pull/38

Reply:
How is that relevant? The bot will stop at cudnn PR.

Reply:
If on these platforms cudnn is not yet available, would the migrator work (in non cudnn feedstocks)?

Reply:
yes

Reply:
When is this migrator supposed to run? I don't see ppc64le build for ptxcompiler.

Reply:
@kkraus14 and I saw that manually applying the migrator would fail, likely because of a conflict with the global pinning (not sure): https://github.com/conda-forge/cub-feedstock/pull/17#issuecomment-1082005780

Reply:
Probably best to move any additional discussion into a new issue

Reply:
See #2679.